scripts/filter_loop.py

```
@@ -44,13 +44,26 @@
+        return val if val > 0 else 2
+    except Exception:
+        return 2
+
+raw_interval = os.environ.get("SH_FILTER_INTERVAL_SEC", "600").strip()
 try:
-    TOP = int(raw_top)
-    if TOP <= 0:
-        raise ValueError(f"SH_TOP_MARKETS must be positive, got {TOP}")
+    INTERVAL_SEC = int(raw_interval)
+    if INTERVAL_SEC <= 0:
+        raise ValueError(f"SH_FILTER_INTERVAL_SEC must be positive, got {INTERVAL_SEC}")
```

Ruff (0.16.2)

[warning] Abstract `raise` to an inner function

(TRY301)